### PR TITLE
NOISSUE - Fix Docker Algorithm

### PR DESCRIPTION
**Pull request by rodneyosodo**
# What type of PR is this?

This is a bug fix PR because it fixes how to run the docker algorithm

## What does this do?

Refactor running or docker algorithm and provide terminal session.

## Which issue(s) does this PR fix/relate to?

No issue

## Have you included tests for your changes?

No tests needed

## Did you document any new/modified features?

Yes

### CODE REVIEW

**Review by danko-miladinovic — APPROVED** [review]
LGTM

File: docs/algorithms.md [hunk 1]
```diff
@@ -529,32 +515,37 @@ cd ./cocos
...
 MANAGER_QEMU_OVMF_VARS_FILE=/usr/share/edk2/x64/OVMF_VARS.fd \
-go run ./cmd/manager/main.go
+go run main.go
 ```
 
 Export the agent grpc url from computation server logs
 
 ```bash
-export AGENT_GRPC_URL=localhost:6015
+export AGENT_GRPC_URL=localhost:6100
```
Comment (WashingtonKK) [inline docs/algorithms.md:542]: Add an indication of what would happen if the port is not avaiable. Or indicate why start with 6100.

File: docs/algorithms.md [hunk 2]
```diff
@@ -569,12 +560,29 @@ After some time when the results are ready, you can run the following command to
 ./build/cocos-cli results ./private.pem
 ```
 
+The logs will be similar to this:
+
+```bash
+2024/08/19 14:14:31 Retrieving computation result file
+2024/08/19 14:14:31 Computation result retrieved and saved successfully!
```
Comment (WashingtonKK) [inline docs/algorithms.md:567]: Add some simple debugging guides. i.e., what error indicates the wrong gRPC port was specified and how to solve it.

**Review by WashingtonKK — CHANGES_REQUESTED** [review]

File: docs/algorithms.md (hunk 1 above)
Comment (rodneyosodo) [reply docs/algorithms.md:542]: I don't think this should be covered here since we don't cover edge cases

File: docs/algorithms.md (hunk 2 above)
Comment (rodneyosodo) [reply docs/algorithms.md:567]: I don't think this should be covered here since we don't cover edge cases. Documentation is currently based on happy flow

Comment (SammyOina) [reply docs/algorithms.md:567]: agreed debugging guide is in the developer guide.

**Review by SammyOina — APPROVED** [review]

**Pull request merged**